src/pages/auth/JoinPage.tsx

```
@@ -22,88 +20,87 @@ export default function SignupPage() {
-    if (code === "123456") {
-      setCodeVerified(true);
-      setCodeError("");
-      clearErrors("code");
-    } else {
-      setCodeVerified(false);
-      setCodeError("인증번호가 일치하지 않습니다.");
-      setError("code", {
-        type: "manual",
-        message: "인증번호가 일치하지 않습니다.",
-      });
+  const checkCode = () => {
```

ESLint

[error] 'checkCode' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)